### PR TITLE
Show a better error message when podman info fails during a refresh

### PR DESCRIPTION
**Pull request by kirbyfan64**

### CODE REVIEW

File: libpod/runtime.go [hunk 1]
```diff
@@ -824,7 +824,11 @@ func (r *Runtime) refreshRootless() error {
 	// Take advantage of a command that requires a new userns
 	// so that we are running as the root user and able to use refresh()
 	cmd := exec.Command(os.Args[0], "info")
-	return cmd.Run()
+	err := cmd.Run()
+	if err != nil {
+		return errors.Wrapf(err, "Error running %s info", os.Args[0])
```
Comment (mheon) [inline libpod/runtime.go]: Maybe "Error running %s info while refreshing state"?

**Comment by mheon** [comment]
bot, add author to whitelist
/approve

**Comment by rhatdan** [comment]
LGTM, with mheons'request change.

**Comment by rhatdan** [comment]
LGTM, with mheons'request change.
Thanks @kirbyfan64 for the PR.

**Comment by baude** [comment]
@kirbyfan64 thank you for your contribution

**Pull request merged**